### DISCUSSION TITLE
{QUESTION/SUGGESTION} Wiping tears

### DISCUSSION THREAD

**Discussion by ELCACAMORA** [Monika-After-Story/MonikaModDev#8987] [QA - code and content creation (technical)]
Idk if I chose the good category, but I'm new with this website so sorry in advance. I thought it would be a good idea that when Monika starts to cry (for X reason) you have the option to do nothing or gently wipe her tears off. She would then have a surprised look and then a wholesome and cute reaction 😌🥺

Sadly I don't know anything about mods and submods so it's really just a suggestion for the devs, or the submods creators 😅

Thank you for taking time to read this. (I Hope it's not already in the mod you just need to have a certain level of affection to have the option cause that would be awkward for me... 😬😵‍💫)

**Comment by M1ntyX0X** [comment]
That honestly would be cutee!!!
